ZeeguuAPI/ZeeguuAPITests/ZeeguuAPITests.swift

```diff
@@ -320,6 +320,58 @@ class ZeeguuAPITests: XCTestCase {
 		testLock.wait()
 	}
 	
+	func testGetTranslationsForWord() {
+		print("Testing getting translations for word:")
+		ZeeguuAPI.sharedAPI().getTranslationsForWord("Gipfeltreffen", context: "Unmittelbar vor dem Gipfeltreffen der Europäischen Union mit der Türkei spricht Parlamentspräsident Martin Schulz (SPD) Klartext - eine Vereinbarung von Flüchtlingskontingenten mit der Türkei sei kaum aussichtsreich.", url: "http://www.spiegel.de/politik/ausland/eu-tuerkei-gipfel-streit-um-fluechtlingskontingent-a-1065093.html") { (dict) -> Void in
+			
+			XCTAssertNotNil(dict)
+			print("translation: ", dict)
```

Comment:
Yes, I could, in fact, the most recent commit does that. However, I have not yet done this for all other endpoints though.

Also, be aware that if something changed (being something deleted or a string edited) in the API, it will break these tests (which is a good thing, but still).